website/content/en/status/report-2023-04-2023-06/freebsd-foundation.adoc

```diff
@@ -73,29 +74,29 @@ Some other partnership-related activities this quarter:
 * I am also pursuing grant opportunities with bodies including:
 ** NSF Secure and Trustworthy Cyberspace (SaTC)
 ** Sovereign Tech Fund
-** NGI
+** NGI.
 
 In terms of research, my broad aim is to make sure that all of the expertise in this community is reflected in the global conversations on computing performance, security, and energy efficiency.
 As a community, we have much to bring to this work.
 
 So far, I have been tracking and plugging into the following threads:
 
 * link:https://openforumeurope.org/open-source/[Open Forum Europe]
-* link:https://www.nist.gov/chips/research-and-development-program[CHIPS Research and Development]
+* link:https://www.nist.gov/chips/research-and-development-program[CHIPS Research and Development].
 
 If you have research ideas or are interested in working together in this area, please link:mailto:greg@freebsdfoundation.org[send me an email], or link:https://calendly.com/greg-freebsdfound/30min[schedule a call].
 
 ==== OS Improvements
 
-During the second quarter of 2023, 339 src, 155 ports, and 20 doc tree commits identified the FreeBSD Foundation as a sponsor.
+During the second quarter of 2023, 339 src, 155 ports, and 20 doc tree commits identified The FreeBSD Foundation as a sponsor.
 Some of this and other Foundation-sponsored work is described in separate report entries:
 
-* Continuous Integration
-* FreeBSD as a Tier 1 cloud-init Platform
+* Continuous integration
```

Comment:
@lwhsu hi, you were a named contact for the first quarter's report, will there be a report for this (second) quarter?